test/dummy_app/test/unit/features_helper_test.rb

```diff
@@ -1,10 +1,11 @@
 require File.expand_path('../../test_helper', __FILE__)
 require 'arturo/features_helper'
+require 'rails-dom-testing'
 
 class ArturoFeaturesHelperTest < ActiveSupport::TestCase
 
   include ActionView::Helpers::TagHelper
-  include ActionDispatch::Assertions::SelectorAssertions
+  include Rails::Dom::Testing::Assertions::SelectorAssertions
```

Comment:
`ActionDispatch::Assertions::SelectorAssertions` is extracted into the `rails-dom-testing` gem in Rails 4.2.